recipes/assimp/5.0.0/conanfile.py

```diff
@@ -0,0 +1,152 @@
+        "with_collada_exporter": "ASSIMP_BUILD_COLLADA_EXPORTER", 
+        "with_csm": "ASSIMP_BUILD_CSM_IMPORTER",  
+        "with_dxf": "ASSIMP_BUILD_DXF_IMPORTER",  
+        "with_fbx": "ASSIMP_BUILD_FBX_IMPORTER", 
+        "with_fbx_exporter": "ASSIMP_BUILD_FBX_EXPORTER", 
+        "with_gltf": "ASSIMP_BUILD_GLTF_IMPORTER", 
+        "with_gltf_exporter": "ASSIMP_BUILD_GLTF_EXPORTER",
+        "with_hmp": "ASSIMP_BUILD_HMP_IMPORTER",  
+        "with_ifc": "ASSIMP_BUILD_IFC_IMPORTER", 
+        "with_irr": "ASSIMP_BUILD_IRR_IMPORTER",  
+        "with_irrmesh": "ASSIMP_BUILD_IRRMESH_IMPORTER",  
+        "with_lwo": "ASSIMP_BUILD_LWO_IMPORTER",  
+        "with_lws": "ASSIMP_BUILD_LWS_IMPORTER", 
+        "with_md2": "ASSIMP_BUILD_MD2_IMPORTER", 
+        "with_md3": "ASSIMP_BUILD_MD3_IMPORTER", 
+        "with_md5": "ASSIMP_BUILD_MD5_IMPORTER", 
+        "with_mdc": "ASSIMP_BUILD_MDC_IMPORTER", 
+        "with_mdl": "ASSIMP_BUILD_MDL_IMPORTER", 
+        "with_mmd": "ASSIMP_BUILD_MMD_IMPORTER", 
+        "with_ms3d": "ASSIMP_BUILD_MS3D_IMPORTER",  
+        "with_ndo": "ASSIMP_BUILD_NDO_IMPORTER", 
+        "with_nff": "ASSIMP_BUILD_NFF_IMPORTER",  
+        "with_obj": "ASSIMP_BUILD_OBJ_IMPORTER", 
+        "with_obj_exporter": "ASSIMP_BUILD_OBJ_EXPORTER", 
+        "with_off": "ASSIMP_BUILD_OFF_IMPORTER",
+        "with_ogre": "ASSIMP_BUILD_OGRE_IMPORTER",
+        "with_opengex": "ASSIMP_BUILD_OPENGEX_IMPORTER",  
+        "with_opengex_exporter": "ASSIMP_BUILD_OPENGEX_EXPORTER",  
+        "with_ply": "ASSIMP_BUILD_PLY_IMPORTER", 
+        "with_ply_exporter": "ASSIMP_BUILD_PLY_EXPORTER", 
+        "with_q3bsp": "ASSIMP_BUILD_Q3BSP_IMPORTER",
+        "with_q3d": "ASSIMP_BUILD_Q3D_IMPORTER",
+        "with_raw": "ASSIMP_BUILD_RAW_IMPORTER",
+        "with_sib": "ASSIMP_BUILD_SIB_IMPORTER",
+        "with_smd": "ASSIMP_BUILD_SMD_IMPORTER",
+        "with_step": "ASSIMP_BUILD_STEP_IMPORTER",
+        "with_step_exporter": "ASSIMP_BUILD_STEP_EXPORTER",
+        "with_stl": "ASSIMP_BUILD_STL_IMPORTER",
+        "with_stl_exporter": "ASSIMP_BUILD_STL_EXPORTER",
+        "with_terragen": "ASSIMP_BUILD_TERRAGEN_IMPORTER", 
+        "with_x": "ASSIMP_BUILD_X_IMPORTER",
+        "with_x_exporter": "ASSIMP_BUILD_X_EXPORTER",
+        "with_x3d": "ASSIMP_BUILD_X3D_IMPORTER",
+        "with_x3d_exporter": "ASSIMP_BUILD_X3D_EXPORTER",
+        "with_xgl": "ASSIMP_BUILD_XGL_IMPORTER",
+    }
+    options.update(dict.fromkeys(format_option_map, [True, False]))
+    default_options.update(dict.fromkeys(format_option_map, True))
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        if not self.options.internal_irrxml:
+            # Using requirement from conan-center
+            self.requires.add("IrrXML/1.2@conan/stable")
```

Comment:
```suggestion
            self.requires.add("IrrXML/1.2@conan/stable")
```
irrxml is not available in CII yet, so you need to wait. Packages from Bincrafters and Conan Commnunity won't be accepted.